src/test/groovy/graphql/nadel/NadelE2ETest.groovy

```diff
@@ -27,7 +28,7 @@ class NadelE2ETest extends Specification {
          }
         '''
 
-    def simpleUnderlyingSchema = TestUtil.schema('''
+    def simpleUnderlyingSchema = typeDefinitions('''
```

Comment:
We no longer get a GraphqlSchema back from underlying tests, but rather type defs - hence all fix ups